### ISSUE TITLE
[css-backgrounds-4] background-position-x and background-position-y should be interpolable

### ISSUE THREAD

**Issue by graouts** [w3c/csswg-drafts#2788]
The current draft of CSS Backgrounds and Borders Module Level 4 defines the [background-position-x](https://drafts.csswg.org/css-backgrounds-4/#propdef-background-position-x) and [background-position-y](https://drafts.csswg.org/css-backgrounds-4/#propdef-background-position-y) properties with their "Animation type" set to "discrete" which means these properties cannot be interpolated. That seems like a mistake.

@birtles @stephenmcgruer @flackr

**Comment by birtles** [comment]
Yes, I notice we seem to animate it as a list of positions in Gecko.

**Comment by flackr** [comment]
Yes, especially since in [css-backgrounds-3](https://drafts.csswg.org/css-backgrounds-3/#background-position) the short-hand `background-position` is animatable.

**Comment by SebastianZ** [comment]
That was already fixed in https://github.com/w3c/csswg-drafts/commit/851531de437c6f95453b77ca147e7da314e6c329.

Sebastian
